src/xUnitToJUnit/xUnitToJUnit.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
Can you please remove `net5.0` from the list of targeted frameworks? It is unsupported since May 10, 2022. Now that we're targeting another Standard Term Support with `net7.0`, we don't need to support `net5.0` anymore. You'll need to do the same for the test project.

I'm planning on keeping `netcoreapp3.1` for a while to let people time to upgrade to .NET 6.